portfolio/src/main/webapp/script.js

```diff
@@ -12,83 +12,122 @@
-
-    /* Changes the styling to active for the tab the user just clicked on */
-    event.currentTarget.className += " active";
+/**  
+ * Displays certain Sections will when a specific tab is 
+ * selected at the top". 
+ * @param {event} arg The event that triggers the website to display
+ *  section-content element.
+ * @param {sectionName} arg This is the sectionName that is used 
+ *  to determine which section to display. 
+ */
+displaySection = (event, sectionName) => {
+  setToNone("section-content");
+  replaceClassName("section-tab", " active");
+  // Displays the appropriate content for the sectionName 
+  // associated with the tab the user just clicked on. 
+  document.getElementById(sectionName).style.display = "block";
+  // Changes the styling to active for the tab the user just clicked on.
+  event.currentTarget.className += " active";
 }
 
-/* Set all elements of given className name to style.display = none*/
-function setToNone(name){
-    var i, content;
-    content = document.getElementsByClassName(name);
-    for(i = 0; i < content.length; i++){
-        content[i].style.display = "none";
-    }
+/** 
+ * Set all elements of given className name to style.display = none.
+ * @param {name} arg The name of the class that I need to modify.
+ */
+setToNone = (name) => {
+  const content = document.getElementsByClassName(name);
+  for (let i = 0; i < content.length; i++) {
+    content[i].style.display = "none";
+  }
+  // Using forEach made the sections no longer display even when 
+  // I clicked on a tab 
+
+  /* content.forEach((element) => {
+    element.style.display = "none";
+  }) */
 }
 
-/*Sets className of all elements of a given className name from "active" to default ""*/
-function setToDefault(name, toRemove){
-    var i, elements;
-    elements = document.getElementsByClassName(name);
-    for(i = 0; i < elements.length; i++){
-        elements[i].className = elements[i].className.replace(toRemove, "");
-    }
+/**
+ * Sets className of all elements of a given 
+ * className name from "active" to default "".
+ * @param {name} arg The name of the class I need to modify.
+ * @param {toRemove} arg The string that I need to remove 
+ *  from the class name I'm modifying.
+ */
+replaceClassName = (name, toRemove) => {
+  const elements = document.getElementsByClassName(name);
+  for (let i = 0; i < elements.length; i++) {
+    elements[i].className = elements[i].className.replace(toRemove, "");
+  }
+  // Using forEach made the sections no longer display even when 
+  // I clicked on a tab
+
+  /* elements.forEach((element) => {
+    element.className = element.className.replace(toRemove, "");
+  }) */
 }
 
-/*Changes the current slide in a slideshow by incrementing by the given n*/
-function changeSlide(n){
-    showSlides(slideIndex += n);
+/** 
+ * Changes the current slide in a slideshow 
+ * by incrementing by the given n. 
+ * @param {n} arg The increment/decrement value for showing the next slide.
```

Comment:
Here and elsewhere, this param could have better naming that reflects its purpose.